src/components/AddLayerModal/index.less

```
@@ -8,4 +22,12 @@
     padding-top: 0.5rem;
   }
 
+  .ant-typography.error {
+    color: red;
+  }
+
+  .url-input {
+    padding: 10px 0;
```

```suggestion
    padding: 1em 0;
```
untested, but relative units (like `em`, `rem` etc.) should be preferred.

---

Thank you! This is now tested and approved